VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: AN400ECU Third_Party_Device

BO_ 218099784 MSGID_0X0CFFF048: 8 AN400ECU
 SG_ RPM : 7|16@0+ (1.0,0) [0|0] "rpm"  Third_Party_Device

BO_ 218101064 MSGID_0X0CFFF548: 8 AN400ECU
 SG_ Battery_Voltage : 7|16@0- (0.01,0) [0|0] "Volts"  Third_Party_Device
 SG_ Air_Temp : 23|16@0- (0.1,0) [0|0] "C/F"  Third_Party_Device
 SG_ Coolant_Temp : 39|16@0- (0.1,0) [0|0] "C/F"  Third_Party_Device

BO_ 218101320 MSGID_0X0CFFF648: 8 AN400ECU
 SG_ Thermistor1_Temp : 7|16@0- (0.1,0) [0|0] "C/F"  Third_Party_Device
 SG_ Thermistor2_Temp : 23|16@0- (0.1,0) [0|0] "C/F"  Third_Party_Device

